contracts/CompoundGovernor.sol

```diff
@@ -359,4 +360,17 @@ contract CompoundGovernor is
     {
         return GovernorTimelockCompoundUpgradeable.state(_proposalId);
     }
+
+    /// @inheritdoc GovernorCountingFractionalUpgradeable
+    // solhint-disable-next-line func-name-mixedcase
+    function COUNTING_MODE() public pure virtual override(IGovernor, GovernorCountingFractionalUpgradeable) returns (string memory) {
+        return "support=bravo,fractional&quorum=for&params=fractional";
+    }
+
+    /// @inheritdoc GovernorUpgradeable
+    /// @dev We override this function to resolve ambiguity between inherited contracts.
```

Comment:
we should change the comment - we override in order to change the quorum strategy. we should also check against the GovernorUpgradeable docs to make sure the natspec is correct (aka that it doesn't reference for + abstain) and if it does, we should not inherit the natspec

Reply:
Done: [9bcc3a0](https://github.com/ScopeLift/compound-governance-upgrade/pull/86/commits/9bcc3a0e0f2485eafbe538a94329674f276f0cbc)